control/controlclient/map.go

```diff
@@ -366,16 +365,11 @@ var (
 	patchifiedPeerEqual = clientmetric.NewCounter("controlclient_patchified_peer_equal")
 )
 
-// updatePeersStateFromResponseres updates ms.peers and ms.sortedPeers from res. It takes ownership of res.
+// updatePeersStateFromResponseres updates ms.peers and ms.sortedPeers from res.
```

Comment:
`ms.sortedPeers` is gone and I think this line also needs to be updated to mention `resp`.